rmw_fastrtps_cpp/src/rmw_event.cpp

```diff
@@ -52,14 +52,14 @@ rmw_ret_t
 rmw_event_set_events_executor_callback(
   const void * executor_context,
```

Comment:
why no remain this to executor_ptr or executor_handle? 
I'm not sure what context means here